docs/deploying.md

```diff
@@ -369,6 +369,150 @@ support.
+          build: "nginx"
+          ports:
+            - "5441:5441"
+          links:
+            - registryv2:registryv2
+        registryv2:
+          build: "../../"
+          ports:
+            - "5000"
+
+ This configuration builds a new `nginx` image as specified by the
+ `nginx/Dockerfile` file. The `registryv2` image is built from the
+ `distribution/Dockerfile` you've used previously.
+	
+3. Build `nginx` and the registry 2.0 image.
+
+        $ docker-compose build
+        Building registryv2...
+        Step 0 : FROM golang:1.4
+        1.4: Pulling from golang
+        ...
+        Step 6 : COPY test.passwd /etc/nginx/test.passwd
+         ---> d6994dfe21b7
+        Removing intermediate container 31fcbfc0402e
+        Successfully built d6994dfe21b7
+
+4. Start both the nginx and registry server with compose.
+
+        $ docker-compose up
+        Creating basicauth_registryv2_1...
+        Creating basicauth_nginx_1...
+        Attaching to basicauth_registryv2_1, basicauth_nginx_1
+        registryv2_1 | time="2015-05-31T04:54:07.598983831Z" level=info msg="endpoint local-8082 disabled, skipping" environment=development instance.id=8206ce7e-f52d-44cf-90c1-aeec0c070457 service=registry version=v2.0.0-alpha.3-332-gc07cca3 
+        registryv2_1 | time="2015-05-31T04:54:07.60037357Z" level=info msg="endpoint local-8083 disabled, skipping" environment=development instance.id=8206ce7e-f52d-44cf-90c1-aeec0c070457 service=registry version=v2.0.0-alpha.3-332-gc07cca3 
+        registryv2_1 | time="2015-05-31T04:54:07.60042165Z" level=info msg="using redis blob descriptor cache" environment=development instance.id=8206ce7e-f52d-44cf-90c1-aeec0c070457 service=registry version=v2.0.0-alpha.3-332-gc07cca3 
+        registryv2_1 | time="2015-05-31T04:54:07.600524214Z" level=info msg="listening on [::]:5000" environment=development instance.id=8206ce7e-f52d-44cf-90c1-aeec0c070457 service=registry version=v2.0.0-alpha.3-332-gc07cca3 
+
+5. In another terminal, verify both servers are running:
+
+        $ docker ps
+        CONTAINER ID        IMAGE                         COMMAND                CREATED              STATUS              PORTS                                     NAMES
+        a4aecbfaebc2        basicauth_nginx:latest        "nginx -g 'daemon of   About a minute ago   Up About a minute   80/tcp, 443/tcp, 0.0.0.0:5441->5441/tcp   basicauth_nginx_1        
+        4f4aeba1db03        basicauth_registryv2:latest   "registry cmd/regist   About a minute ago   Up About a minute   0.0.0.0:32768->5000/tcp                   basicauth_registryv2_1  
+  
+### Test basic authentication on the registry
+
+1. Log into the registry with the test user.
+
+        $ docker login -u testuser -p passpassword -e distribution@docker.com localhost:5441
+        WARNING: login credentials saved in /home/ubuntu/.dockercfg.
+        Login Succeeded
+
+2. Tag the `nginx` 1.9 image.
+
+        $ docker tag nginx:1.9 localhost:5441/nginix:1.9
```

Comment:
Is it deliberate to tag as `nginix` (with an extra 'i'), or just a typo? It confused me.